### ISSUE TITLE
El Dorado can't override deliverycharge

### ISSUE THREAD

**Issue by kbower** [kbower/tickettest1#1302]
_Imported from trac ticket 1302.
Created by **kbower**
Opened in trac: Dec 27, 2011, 11:52:47 AM
Last modified in trac: Jan 9, 2012, 11:47:00 AM_
***
Kent,

I’m OK with it recalculating the delivery when something changes, but it does it all the time.

I tried changing the delivery charge for order 1226120PDDW without changing anything else and it reverted back to the $25 dollars.

Any suggestions?

Mariam

**Comment by kbower** [comment]
_**kbower** trac comment on Jan 7, 2012, 11:27:52 AM:_

We need to be more picky about reset of deliverycharge.  
`customerorder.totaldiscount`
`status`
```python
reset_value_dependencies('deliverycharge', Order,
    'customerorder.totaldiscount','shipzoneid','deliverytype','orderlines.qtyordered',
        'orderlines.itemprice','orderlines.voided','deliverydate','status')
```

**Comment by kbower** [comment]
_**kbower** trac comment on Jan 9, 2012, 11:47:00 AM:_

(Changeset [[2126]](../commit/de4ad4fba80a4c3e081088c4e456c8f111ee063a))
 * fixes #1302 (El Dorado can't override deliverycharge)

**Comment by kbower** [comment]
_**kbower** trac status change on Jan 9, 2012, 11:47:00 AM:_ **closed**
